[DOCS-5544] adding clean up old Jenkins build video (#5142)
jenkins-infra/jenkins.io: content/doc/book/pipeline/pipeline-best-practices.adoc

Solution: The best solution would be to work around the calls being made, but if they must be done then it would be better to implement a Jenkins plugin which is able to gather the data needed.

=== Cleaning up old Jenkins builds

As a Jenkins administrator, removing old or unwanted builds keeps the Jenkins controller running efficiently.

> When older builds are not discarded, resources are taken away from more current and relevant builds.

Reviewer: ```suggestion
When you do not remove older builds, there are less resources for more current and relevant builds.
```

> This video reviews using the `build.Discarder` directive in a Pipeline for Global and Specific usage.

Reviewer 2: Use the correct spelling of the declarative Pipeline directive.  Link to the more detailed documentation.
```suggestion
This video reviews using the link:/doc/book/pipeline/syntax/#options[`buildDiscarder`] directive in individual Pipeline jobs.
```

> Additionally, the video presents keeping specific historical builds.

Reviewer: ```suggestion
The video also reviews the process to keep specific historical builds.
```
